azavea/raster-foundry
# Bucket editing / Color correction

## Overview

This adds functionality to the Bucket edit button, and also sets up a color correction tool with stubbed histogram update methods.

### Checklist


### Demo

Note: There is a known styling issue with the sidebar not filling up the whole screen, which causes the map not to fill the screen either, which is what's going on in the screenshot below. I'm going to work with @designmatty tomorrow to fix this.

![color correction](https://cloud.githubusercontent.com/assets/447977/20156442/51c69ade-a69e-11e6-8ab8-57464f69c61a.png)

### Notes
This includes more line-ending fixes; as before, I've kept these as a separate commit so that they're easier to review.

## Testing Instructions

 * From inside VM, `./scripts/server`
 * From project root, `cd app-fronted && npm start`
 * Log in to `http://localhost:9091` and use the UI to navigate to <username> -> My Library -> Buckets -> Click to view an existing bucket or create a new one.
 * Click the "Editor" link on the upper right side of the page
 * Note that you are taken to a list of the bucket's scenes.
 * Click any scene and note that you are taken to a color correction panel
 * Drag the histogram adjustment sliders and note that the placeholder histogram adjusts.

Connects #626, closes #625
